lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +142,23 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+          options[:color_mode] = :automatic
+        end
```

Comment:
Woops, somehow I didn't get notified about this comment or just didn't notice the email or something.  Sorry about that!



> I think backwards compatibility has a higher priority than the clarity of where it is in the source code.

Agreed.

> Maybe there is another way to make it clearer that it won't be listed in the output. A comment? A change to the text "description" passed into parser.on?

Both of those would help.  The description is partly what threw me originally: since you took the time to update it to `Deprecated: ...` it made it seem like it would be printed.  Replacing it with `''` and a comment to explain it would address the issue well enough, I think.